.idea/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Default ignored files
```

Comment:
In global .gitignore ignore .idea for all projects. https://stackoverflow.com/a/7335487